src/components/book.js

```diff
@@ -1,11 +1,39 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
Disabling of these rules appear to be unnecessary, let's fix the codebase instead.

Reply:
DONE